modules/ipp/scanner.go

```diff
@@ -0,0 +1,206 @@
+// Package ipp provides a zgrab2 module that scans for ipp.
+// TODO: Describe module, the flags, the probe, the output, etc.
+package ipp
+
+//TODO: Clean up these imports
+import (
+	//"bytes"
+	"encoding/binary"
+	//"errors"
+	//"fmt"
+	//"io"
+	"net/http"
+	"strconv"
+	"strings"
+	//"net"
+	//"net/url"
+	//"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/zmap/zgrab2"
+)
+
+const (
```

Comment:
Go constants are typically of the form `ContentType` rather than `CONTENT_TYPE`; the golint will warn on the latter.

If the constants are copied from another API or an RFC, then keeping the well-known names is more important than language style, but for internal constants / constants that we're defining, it's probably best to use the go style.

 - [ ] `CONTENT_TYPE` -> `ContentType`